### PR TITLE
[stable/k8s-spot-rescheduler] bump k8s-spot-rescheduler to v0.2.0

### PR DESCRIPTION
**Pull request by kimxogus**
**What this PR does / why we need it**: Bump app version to `v0.1.2`

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by mattfarina** [comment]
Waiting for the charts maintainer to respond.

**Comment by komljen** [comment]
Looks good to me.

File: stable/k8s-spot-rescheduler/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: "v0.1.1"
+appVersion: "v0.1.2"
```
Comment (komljen) [inline stable/k8s-spot-rescheduler/Chart.yaml]: There is a new version available, so we could probably use the latest one https://github.com/pusher/k8s-spot-rescheduler/releases/tag/v0.2.0

File: stable/k8s-spot-rescheduler/values.yaml [hunk 2]
```diff
@@ -5,20 +5,22 @@ replicaCount: 1
 
 image:
   repository: quay.io/pusher/k8s-spot-rescheduler
-  tag: v0.1.1
+  tag: v0.1.2
```
Comment (komljen) [inline stable/k8s-spot-rescheduler/values.yaml]: Here also, the version v0.2.0 is available.

File: stable/k8s-spot-rescheduler/Chart.yaml (hunk 1 above)
Comment (kimxogus) [reply stable/k8s-spot-rescheduler/Chart.yaml]: updated to v0.2.0 with new clusterrole :)

**Comment by kimxogus** [comment]
/assign @mattfarina 
seems seanknox is not notifying this PR
can you review review this?

**Comment by kimxogus** [comment]
seems /ok-to-test is needed

**Pull request merged**